CMakeLists.txt

```diff
@@ -879,8 +890,16 @@ foreach (testScript ${TEST_SCRIPTS})
         ENVIRONMENT "parser_skip=skip"
     )
   endif()
+
 endforeach()
 
+# skip parallel netcdf tests if its not enabled in given install
+IF(NOT NetCDF_PARALLEL)
```

Comment:
Leaving what @J-Lentz pointed out, line 290 should say "parallel netcdf io" instead of "collective netcdf io"